pmm-app/tests/QAN/filters_test.js

```diff
@@ -223,3 +222,75 @@ Scenario(
     }
   },
 );
+
+Scenario(
+  'PMM-T191 - Verify Reset All and Show Selected filters @not-pr-pipeline @qan',
+  async (I, qanPage, qanActions) => {
+    const environmentName1 = 'ps-dev';
+    const environmentName2 = 'pgsql-dev';
+    qanActions.waitForNewQANPageLoaded();
+    qanActions.applyFilterNewQAN(environmentName1);
+    qanActions.applyFilterNewQAN(environmentName2);
+    I.click(qanPage.fields.showSelected);
+    await qanActions.verifyCountOfFilterLinks(2, false);
+    I.click(qanPage.fields.resetAllButton);
+    await qanActions.verifyCountOfFilterLinks(2, true);
+
+    qanActions.applyFilterNewQAN(environmentName1);
+    I.click(qanPage.fields.showSelected);
+    await qanActions.verifyCountOfFilterLinks(1, false);
+    qanActions.applyFilterNewQAN(environmentName1);
+    await qanActions.verifyCountOfFilterLinks(1, true);
+  },
+);
+
+Scenario(
+  'PMM-T190 - Verify user is able to see n/a filter @not-pr-pipeline @qan',
+  async (I, qanPage, qanActions) => {
+    qanActions.waitForNewQANPageLoaded();
+    I.fillField(qanPage.fields.filterBy, 'n/a');
+    await qanActions.verifyCountOfFilterLinks(0, true);
```

Comment:
what exactly are we trying to achieve here?

Reply:
This test case: https://jira.percona.com/secure/Tests.jspa#/testCase/375 :) After I have filtered by "n/a" I am checking that count of filters isn't 0. :)